string ugv_id
float32 height
